### ISSUE TITLE
Hello all, I am working on EEGLOGGER example. I have received an error while running on the example. "Received: Internal JSON-RPC error. Message Error recieved from CortexAccess.CortexClient Recieved: -32603 : Internal JSON-RPC error."  Please help.  

### ISSUE THREAD

**Issue by chetanvijayin** [Emotiv/cortex-example#27]
Operating system: ✍️ TODO
Last upstream commit (run `git log --author="Robert Lord" | head -n 1`): ✍️ TODO
language version:(nodejs version, ruby version ....) ✍️ TODO
Cortex version:  ✍️ TODO

---

✍️ TODO write your issue here

**Comment by patdotme** [comment]
@chetanvijayin can you please describe how to reproduce the error that you saw please?

**Comment by chetanvijayin** [comment]
Hi @patdotme, I am using Mac. Here is the steps I followed:

1. I downloaded the file from https://github.com/Emotiv/cortex-example 

2. I used Visual Studio to Open the project "EEGLOGGER"

3. I inserted the Username, Password and LicenseID in "Program.cs".

4. I used Nuget Package Manager to install _Newtonsoft.Json, SuperSocket.ClientEngine.Core, WebSocket4Net_ for CortexAccess project

5. The project was build successfully without any errors but with one warning message - /Library/Frameworks/Mono.framework/Versions/5.10.1/lib/mono/msbuild/15.0/bin/Microsoft.Common.CurrentVersion.targets(5,5): Warning MSB3836: The explicit binding redirect on "SuperSocket.ClientEngine, Culture=neutral, PublicKeyToken=ee9af13f57f00acc" conflicts with an autogenerated binding redirect. Consider removing it from the application configuration file or disabling autogenerated binding redirects. The build will replace it with: "<bindingRedirect oldVersion="0.0.0.0-0.10.0.0" newVersion="0.10.0.0" xmlns="urn:schemas-microsoft-com:asm.v1" />". (MSB3836) (EEGLogger) '

6. When I run the code, I get this error message - Received: Internal JSON-RPC error.
Message Error recieved from CortexAccess.CortexClient
Recieved: -32603 : Internal JSON-RPC error.

Thanks
